includes/class-wc-calypso-bridge-ecommerce-admin-menu.php

```diff
@@ -75,6 +77,7 @@ protected function handle_free_trial_menu() {
 
 			// Hide Extensions > Manage.
 			$this->hide_submenu_page( 'woocommerce', 'admin.php?page=wc-addons&section=helper' );
+			$this->hide_submenu_page( 'woocommerce', 'wc-admin&path=/extensions' );
```

Comment:
Maybe we can spell out in the comment `Hide Extensions > Manage and the new Extensions page.`

Reply:
Done in https://github.com/Automattic/wc-calypso-bridge/pull/1318/commits/d4354a1134850435aea931121a16fe828e5dc800